src/protocol/http/src/main/java/org/apache/jmeter/protocol/http/proxy/Proxy.java

```diff
@@ -89,6 +89,19 @@
     private static final String SSLCONTEXT_PROTOCOL =
         JMeterUtils.getPropDefault("proxy.ssl.protocol", "TLS"); // $NON-NLS-1$ $NON-NLS-2$
 
+    private static final String PROTOCOL_LIST =
+            JMeterUtils.getPropDefault("https.socket.protocols", ""); // $NON-NLS-1$ $NON-NLS-2$
+
+    private static final String CIPHER_SUITE_LIST =
+            JMeterUtils.getPropDefault("https.cipherSuites", ""); // $NON-NLS-1$ $NON-NLS-2$
+
+    private static final String[] SUPPORTED_PROTOCOL_LIST =
```

Comment:
The naming looks misleading.
What do you think if `String PROTOCOL_LIST` was named `String HTTPS_SOCKET_PROTOCOLS`, and `String[] SUPPORTED_PROTOCOL_LIST` was `String[] HTTPS_SOCKET_PROTOCOLS_ARRAY`?

On the other hand, it might make sense to add `JMeterUtils.getArrayPropDefault` or something like that since it is probably not the last time we have an array property. It would eliminate the need of non-array field (so there would not be a need to invent a name for it)